### PR TITLE
[UIKit] Update bindings to Xcode 11 beta 4

### PR DESCRIPTION
**Pull request by dalexsoto**

### CODE REVIEW

File: src/uikit.cs [hunk 1]
```diff
@@ -21262,6 +21274,10 @@ interface UITabBarItemAppearance : NSCopying, NSSecureCoding {
 	// 	[Export ("applySnapshot:animatingDifferences:")]
 	// 	void ApplySnapshot (NSDiffableDataSourceSnapshot<SectionIdentifierType, ItemIdentifierType> snapshot, bool animatingDifferences);
 
+	// 	[Async]
```
Comment (chamons) [inline src/uikit.cs]: Commented out for a reason?

Comment (dalexsoto) [reply src/uikit.cs]: @chamons Yes, this needs some registrar changes, the binding will be enabled in a later PR.

Comment (chamons) [reply src/uikit.cs]: Issue please :)

Comment (dalexsoto) [reply src/uikit.cs]: Already here :) https://github.com/xamarin/xamarin-macios/issues/6567

Comment (dalexsoto) [reply src/uikit.cs]: @chamons it was part of the previous UIKit PR, just updated the commented out binding.

Comment (chamons) [reply src/uikit.cs]: Added to before we ship issue.

**Pull request merged**